### PR TITLE
feat(sealos create): Add command preview functionality

### PR DESCRIPTION
**Pull request by LZiHaN**
### Summary
:sparkles::recycle::package:

This pull request enhances the `buildah` package to use the buildah library and improve the container creation process, cleans up the `guest` package and removes unused code, updates the `service/go.work.sum` file with new dependencies, adds new constants for image types to the `constants` package, and adds a utility function for formalizing commands to the `strings` package. These changes aim to improve the functionality, readability, and reusability of the sealos codebase.

> _This pull request has some changes to show_
> _For sealos, buildah, and guest packages, you know_
> _It adds some constants and strings_
> _And cleans up some unused things_
> _And updates the `go.work.sum` for the go_

### Walkthrough
*  Add new imports to use the buildah library, the expansion package, and the v2 image type constants in the `buildah` package ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-58d7cc78400df81b474dbdb004cbae50ccbb6edf8659d6ab34d81eb0b0eb3ae4L22-R28))
*  Add a line to print the shell commands for the container in the `newCreateCmd` function in the `buildah` package ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-58d7cc78400df81b474dbdb004cbae50ccbb6edf8659d6ab34d81eb0b0eb3ae4R95))
*  Define the `printCommands` function to format and expand the entrypoint and cmd commands in the `buildah` package ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-58d7cc78400df81b474dbdb004cbae50ccbb6edf8659d6ab34d81eb0b0eb3ae4R150-R167))
*  Add new constants for different types of images in the `constants` package ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-55fb56b28e5db2ca378563d4d67882d685e3859206ea98a0265b3379138f3791R24-R29))
*  Remove unused imports from the `guest` package ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-2fff8ad69185df503a777c91856e8cf472e5a5d16905d4a3a2e544873e889c46L19), [link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-2fff8ad69185df503a777c91856e8cf472e5a5d16905d4a3a2e544873e889c46L25))
*  Remove the `formalizeWorkingCommand` function from the `guest` package and move it to the `strings` package as `FormalizeWorkingCommand` ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-2fff8ad69185df503a777c91856e8cf472e5a5d16905d4a3a2e544873e889c46L84-L96), [link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-82b41b8126e4fd18dd3847b75fd2060a8b5de00c716db1fd05ce9f14edecd253R237-R249))
*  Modify the `formalizeImageCommands` function in the `guest` package to use the `strings.FormalizeWorkingCommand` function and pass the image type as a string ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-2fff8ad69185df503a777c91856e8cf472e5a5d16905d4a3a2e544873e889c46L104-R97))
*  Modify the `service/go.work.sum` file to add and update dependencies ([link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-592406098feda35460430c3e0bc5ac501ddf1b4e43bb57c8586752868cae787cL1-R104), [link](https://github.com/labring/sealos/pull/4291/files?diff=unified&w=0#diff-592406098feda35460430c3e0bc5ac501ddf1b4e43bb57c8586752868cae787cL41-R133))

### CODE REVIEW

File: pkg/utils/strings/strings.go [hunk 1]
```diff
@@ -234,3 +234,16 @@ func TrimQuotes(s string) string {
 	}
 	return s
 }
+
+func FormalizeWorkingCommand(clusterName string, imageName string, t string, cmd string) string {
```
Comment (fengxsong) [inline pkg/utils/strings/strings.go:238]: `stringsutil` only contains utils related to strings ops. Do not move `formalizeWorkingCommand` into this package.

Comment (LZiHaN) [reply pkg/utils/strings/strings.go:238]: thanks for the feedback! I unsure about the best location for **formalizeWorkingCommand**. any suggestions?

File: pkg/guest/util.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+	"github.com/labring/sealos/pkg/constants"
+)
+
+func FormalizeWorkingCommand(clusterName string, imageName string, t string, cmd string) string {
+	if cmd == "" {
+		return ""
+	}
+	switch t {
+	case constants.RootfsImage, constants.PatchImage:
```
Comment (cuisongliu) [inline pkg/guest/util.go:30]: image
using this define ?

Comment (LZiHaN) [reply pkg/guest/util.go]: done

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**